backend/hdf5/DataArrayHDF5.cpp

```diff
@@ -131,13 +131,13 @@ vector<double> DataArrayHDF5::polynomCoefficients() const {
 }
 
 
-void DataArrayHDF5::polynomCoefficients(const vector<double> &coefficients) {
+void DataArrayHDF5::polynomCoefficients(const vector<double> &coefficients, const Compression &compression) {
     DataSet ds;
     if (group().hasData("polynom_coefficients")) {
         ds = group().openData("polynom_coefficients");
         ds.setExtent({coefficients.size()});
     } else {
-        ds = group().createData("polynom_coefficients", H5T_NATIVE_DOUBLE, {coefficients.size()});
+         ds = group().createData("polynom_coefficients", H5T_NATIVE_DOUBLE, {coefficients.size()}, compression);
```

Comment:
nitpick: did a space smuggle itself in?

Reply:
Also more importantly: you really think it is necessary to compress the polynom dataset? Shouldn't those be only a few entries at all?

Reply:
actually not, that is because the header defines the default to be Compression::None